test/test-mocha.js

```diff
@@ -0,0 +1,5 @@
+const chai = require('chai');
```

Comment:
I'm not sure what's going on with this file.  If it needs to stand alone like this, some comments would be helpful.

Reply:
Test setup files are pretty normal I think the difference here is I called this test-mocha instead of test-node. I will add some comments soon.

Reply:
@mattcollier this was addressed in this commit: https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/4ce66659726981b6ae0b0a82976bf9892d2195c7 which is in the other branch all tests.